tests/test.rb

```diff
@@ -2,8 +2,9 @@
 require 'safe_yaml'
 
 data = SafeYAML.load_file '_data/project_filter.yml', safe: true
-expect_size = data.size
-projectdir = Dir.new('_site/dashboard/project')
+redirects = Dir.open('pages/project').count-2
```

Comment:
Ah, `.` and `..`, right. In that case, perhaps `Dir.glob('pages/project/*')`, which won't match dotfiles.